src/alire/alire-origins-deployers-system-homebrew.adb

```diff
@@ -0,0 +1,219 @@
+with AAA.Strings; use AAA.Strings;
+
+with Alire.OS_Lib.Subprocess;
+with Alire.Errors;
+
+with GNATCOLL.JSON;
+
+package body Alire.Origins.Deployers.System.Homebrew is
+
+   pragma Warnings (Off);
```

Comment:
All warnings off for the entire file? You can target a specific warning, but disabling all of them is not ok.